translib/transformer/test/transformer_test.go

```diff
@@ -0,0 +1,299 @@
+//////////////////////////////////////////////////////////////////////////
```

Comment:
Better to keep it in the transformer directory itself

Reply:
Addressed as suggested